db/models.py

```diff
@@ -50,3 +53,60 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(on_delete=CASCADE, to="User")
```

Comment:
The `User` model is referenced before it is defined. This can lead to confusion and potential errors. It is a good practice to define all models before their usage. Consider moving the `User` model definition before the `Order` model.